docker/grafana/Dockerfile

```diff
@@ -3,9 +3,13 @@ FROM grafana/grafana:latest
 COPY ./docker/grafana/datasource.yaml /etc/grafana/provisioning/datasources/datasource.yaml
 COPY ./docker/grafana/dashboards.yaml /etc/grafana/provisioning/dashboards/all.yaml
 
+COPY ./integrations/grafana/m3db_dashboard.json /tmp/m3db_dashboard.json
+RUN mkdir -p /tmp/grafana_dashboards
```

Comment:
can you just run this command first and avoid the first copy above? Not seeing the point of the two copies. I think what it did before was copy it to tmp, make the new directory, then use awk to create a new file in the new folder which made a little more sense, but right now this first copy seems like basically a no-op

Reply:
also it looks like you deleted the awk command for `m3db_dashboard`, could you make sure that dashboard still works before merging this?

Reply:
oh nevermind I see how it works with the template var now.

Reply:
Yeah, ran the m3db_dashboard and it; I think it defaults to m3query

Good call on the copy, will fix it up